src/devtools/client/debugger/src/components/Editor/ColumnBreakpoint.tsx

```diff
@@ -71,7 +71,7 @@ export default function ColumnBreakpoint({
     });
 
     const bookmark = doc.setBookmark(
-      { line: point.location.line, ch: point.location.column },
+      { line: point.location.line - 1, ch: point.location.column },
```

Comment:
I thought Code Mirror used 1-based lines and 0-based columns?

Reply:
I don't know, all I can tell is that we had this before the refactor and it makes column breakpoints show up on the correct line again.